addons/l10n_in/__manifest__.py

```diff
@@ -50,4 +50,9 @@
         'demo/product_demo.xml',
     ],
     'license': 'LGPL-3',
+    'assets': {
```

Comment:
version above to 2.1?

Reply:
I am afraid because it's an update module in every indian DB in 17.0.
what if there is an error?
because of customization in view or something.